tests/test_admin.py

```diff
@@ -216,3 +218,106 @@ def test_except_cred_list(client):
     assert result.output == (
         f"Credentials listing failed. \n" f"'NoneType' object has no attribute 'user'\n"
     )
+
+
+def fake_clients():
+    user = [
+        {
+            "userId": "jerrygarcia",
+            "fullName": "Jerry Garcia",
+            "emailAddr": "garcia@bgn.net",
+            "address1": "456 Shakedown St.",
+            "city": "Portsmouth",
+            "active": True,
+        },
+        {
+            "userId": "johnthompson",
+            "fullName": "John Thompson",
+            "emailAddr": "",
+            "address1": "",
+            "city": "",
+            "active": True,
+        },
+    ]
+    client = mock.Mock()
+    client.user.list.return_value = user
+    client.user.side_effect = client.user.list().pop(1)
```

Comment:
IMHO, this is not the right way to test. This will make your test case always return True, even though you call the wrong function next (e.g rm), because you do correct the input `pop` before process.